packages/plugins/Traderxyz/src/base.ts

```diff
@@ -0,0 +1,17 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_DESCRIPTION, PLUGIN_ID, PLUGIN_NAME } from './constants'
+import { languages } from './locales/languages'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: '' }, link: '' },
```

Comment:
Please fill those fields.

Reply:
I am using constant to define the variables as seen below.
<img width="492" alt="image" src="https://user-images.githubusercontent.com/3579069/165039681-d1d471b7-dd8b-4f0a-8752-563cdc53288a.png">

Reply:
And you still need to refer it here.

Reply:
Not resolved.